app/services/suggester/users/sidebar.rb

```diff
@@ -14,7 +14,7 @@ def suggest
             where("positive_reactions_count > ?", reaction_count).
             where("published_at > ?", 4.months.ago).
             where("user_id != ?", user.id).
-            where.not(user_id: user.following_by_type("User").pluck(:id)).
+            where.not(user_id: user.following_by_type("User")).
```

Comment:
by default, if no `select()` statement is added, Rails selects primary keys, the ID